SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -904,7 +912,7 @@ int32 USpatialNetDriver::ServerReplicateActors_ProcessPrioritizedActors(UNetConn
 		}
 	}
 
-	return FinalReplicatedCount;
+	return FinalReplicatedCount + FinalCreationCount;
```

Comment:
Investigate how this value is used when returned and indexed into the sorted actor array.